blogpost_14.html

```diff
@@ -0,0 +1,347 @@
+                  <img class="full-img" src="images/blogpost_14/thunar_transfer_dialog.png" alt="thunar - file transfer dialog"/>
+
+                <h3>Launchers</h3>
+                  <p>By default, launcher names instead of filenames are shown now for trusted launchers.</p>
+
+                <h3>Undo/Redo</h3>
+                  <p>The user now will be asked before any deletion will be done by an undo operation.</p>
+
+                <h3>List View</h3>
+                  <p>Similar to the tree-view side pane, the main view now also supports folder expansion.</p>
+                  <img class="full-img" src="images/blogpost_14/thunar_expandable_folders.png" alt="thunar - expandable folders"/>
+
+                <h3>Split View</h3>
+                  <p>A shortcut now can be used to toggle between split-view panes, and the focused pane now uses different coloring.</p>
+                  <img class="full-img" src="images/blogpost_14/thunar_split_view.png" alt="thunar - split view"/>
+                  <p>A dialog will now ask for confirmation when closing a split-pane with multiple tabs.</p>
+            </div>
+
+            <h2><a href="https://docs.xfce.org/xfce/xfce4-appfinder/start" target="_blank">xfce4-appfinder</a></h2>
+            <div class="t1">
+
+                <p>The application list now as well can be navigated via Ctrl+N / Ctrl+P keys (VI default).</p>
+                <p>An option was added to close the Appfinder window automatically when focus is lost.</p>
+                <p>ctrl+enter can be used to execute commands immediately.</p>
+                <p>There is now an option to show generic names of launchers.</p>
+                <p>Appfinder optionally can now be launched as a daemon, so that it will respond even quicker.</p>
+                <p>An option allows now to launch items on a single click.</p>
+
+                <p>Here is a comparison between xfce4-appfinder settings in Xfce4.18 (left) and Xfce4.20 (right).</p>
+                <img class="full-img" src="images/blogpost_14/appfinder_old.png" alt="xfce4-appfinder - settings Xfce 4.18"/>
+                <img class="full-img" src="images/blogpost_14/appfinder_new.png" alt="xfce4-appfinder - settings Xfce 4.18"/>
+
+                <p>Add support to application actions (<a href="https://gitlab.xfce.org/xfce/xfce4-appfinder/-/issues/86">#86</a>)</p>
+                <p class="note">@AndreLDM: I don't see these action .. maybe some missing dependency on my side. Could you please add info on it, if required? (Or should we drop the item?)</p>
+
+                <p>New option to toggle window visibility of background service (<a href="https://gitlab.xfce.org/xfce/xfce4-appfinder/-/issues/52">#52</a>)</p>
+                <p class="note">@AndreLDM: From a quick look I don't get what this one is about .. do we need to list it here?</p>
```

Comment:
This is a new dbus method that allows to open/close appfinder's window. I'm not sure this is relevant to the majority of users, I'll leave it up to you whether to keep this point.

![image](https://github.com/user-attachments/assets/e314801e-ceeb-483b-b748-2d54f0158faf)

Reply:
Ah, ok, thanks for the details ... I guess this is indeed not relevant for the tour, dropping it from the list.